.github/workflows/test_on_push.yml

```diff
@@ -86,4 +87,36 @@ jobs:
         run: python -m pip install nox
 
       - name: Check if the documentation can be built
-        run: python -m nox -s docs
+        run: nox -s docs
+
+  generated_project_tests:
+      needs: [template_test]
```

Comment:
Should be independent imo
```suggestion
```

Reply:
I am guessing, if the `template_test` fails, that is if the template itself fails a generation of a project, then there is no purpose of running tests inside the generated project.

Reply:
Oh oh, I see, yes that makes sense. Thanks!